### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by aryansarkar13**
Events Page

### CODE REVIEW

**Comment by naveennvrgup** [comment]
no need of new  frontend folder for each new feature

File: frontend/src/Startups/startup.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  margin: auto;
+  font-size: 15px;
+  padding-bottom: 60px;
+  color: #fff;
+  width: 50%;
+  
+}
+
+#start {
+  background:url(../Images/start/wp.jpg);
```
Comment (kdpisda) [inline frontend/src/Startups/startup.css:20]: @naveennvrgup sure won't break?

File: frontend/src/Startups/startup.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import s1 from '../Images/start/1.png';
+import s2 from '../Images/start/2.jpg';
+import s3 from '../Images/start/3.jpeg';
+import s4 from '../Images/start/4.jpg';
+import '../Startups/startup.css';
+
+const startup=()=> (
```
Comment (kdpisda) [inline frontend/src/Startups/startup.js:8]: Better make a react component what say @naveennvrgup ?

**Review by kdpisda — CHANGES_REQUESTED** [review]

File: frontend/src/Startups/startup.js (hunk 2 above)
Comment (naveennvrgup) [reply frontend/src/Startups/startup.js:8]: @kdpisda I had a video call with frontend guys to not do this immediately after this commit. explained them how to fetch this from our apis and do this dyanamically the react way.

File: frontend/src/Startups/startup.css (hunk 1 above)
Comment (naveennvrgup) [reply frontend/src/Startups/startup.css:20]: @kdpisda I had a video call with frontend guys to not do this immediately after this commit. explained them how to fetch this from our apis and do this dyanamically the react way.

**Comment by naveennvrgup** [comment]
@kdpisda I had a video call with frontend guys to not do this immediately after this commit. explained them how to fetch this from our apis and do this dyanamically the react way. they r building the components in  react way with the reference i gave them. *please dont reopen this PR as this might create confusion*

**Pull request closed without merging**